include/ringmesh/geomodel/core/geomodel.h

```diff
@@ -306,6 +310,8 @@ namespace RINGMesh
             std::unique_ptr< GeoModelGeologicalEntity< DIMENSION > > > >
             geological_entities_;
 
+        std::unique_ptr< const StratigraphicColumn > strati_column_;

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make this member variable "private". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3656)